fish-rust/src/pager.rs

```diff
@@ -379,6 +380,11 @@ impl Pager {
 
         let needle = self.search_field_line.text();
 
+        // Bypass glob pattern
+        if parse_util_contains_wildcard(needle) {
```

Comment:
this is a leftover (it's wrong for the completion pager).
Also it's no longer necessary, if you squash your commits that should be obvious

Reply:
Fixed in https://github.com/fish-shell/fish-shell/pull/10143/commits/740718c08d26da5f35784af027c2d9c885dd57e9.